cmd/livepeer/livepeer.go

```diff
@@ -134,6 +135,7 @@ func main() {
 	ethKeystorePath := flag.String("ethKeystorePath", "", "Path for the Eth Key")
 	ethOrchAddr := flag.String("ethOrchAddr", "", "ETH address of an on-chain registered orchestrator")
 	ethUrl := flag.String("ethUrl", "", "Ethereum node JSON-RPC URL")
+	maxTxReplacements := flag.Int("maxTransactionReplacements", 0, "Number of times to automatically replace pending Ethereum transactions")
```

Comment:
A feature that changes the existing behaviour should in my opinion not be enabled by default.

Reply:
But I'm open to discuss this, I feel like if replacements should be the default then I think the default should be one replacement, which at least for the reward calls retains the same behaviour.

Reply:
> A feature that changes the existing behaviour should in my opinion not be enabled by default.

Well that depends on whether the existing default behavior is a problem. Based on #1906, I think this is a case where the existing default behavior can actually be a problem. Also, if the default is 0, then we'd be changing existing behavior for reward calls except this is actually hampering the node's ability to deal with getting transactions confirmed in a timely manner for reward calls instead of improving it.

> I feel like if replacements should be the default then I think the default should be one replacement

I'd be onboard with a default of 1 to be a bit more conservative with the default.